Automattic/jetpack
# Sharing Block: add SMS button

Fixes https://github.com/Automattic/jetpack/issues/35426

## Proposed changes:

Adding SMS sharing button

https://github.com/Automattic/jetpack/assets/60262784/5ee1ed08-c0a1-49e0-9c35-22de1a673a71


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up this branch (easiest way to spin up a JN site with Jetpack Beta tester and this branch or simpy use localhost).
* Create a new post and add sharing buttons to it.
* Add sms sharing button. Check that styling is correct and check how it's work on the post.
* There's no way for me to test on a platform other than Mac, so if someone could it would be great.
